packages/dynamic-require/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@theia/dynamic-require",
+  "version": "1.14.0",
```

Comment:
The package states `1.14.0` but the `core` package requires `1.16.0`. Is this the source of the build error?